concepts/sbtc/auxiliary-features/signer-wallet-rotation.md

```diff
@@ -0,0 +1,41 @@
+# Signer Wallet Rotation
+
+Signer Wallet Rotation is a crucial security feature in the sBTC system that allows sBTC Signers to rotate their private keys securely.
+
+## Overview
+
+- sBTC Signers have the ability to rotate their private keys.
```

Comment:
I think this claim needs more context - can we link to a SIP or repo/docs that define how this occurs?

Reply:
Will ink to SIP when it is released, keeping this section of the docs high-level for now.